service_container/parameters.rst

```diff
@@ -243,8 +243,9 @@ Constants as Parameters
 -----------------------
 
 Setting PHP constants as parameters is also supported.
-To take advantage of this feature, map the name of your constant to a parameter
-key and define the type as ``constant``.
+For the XML format, you need to map the name of your constant to a parameter
+key and define the type as ``constant``. For the YAML format, you need to prefix 
+the name of your constant with ``!php/const:``.
```

Comment:
The documentation now says "To take advantage of this feature, map the name of your constant to a parameter key and define the type as ``constant``". But this is true only for XML format. We need to remove this or to specify that is only for XML format and describe the other ones too. I preferred the second one. The PHP format hasn't been described as it is trivial IMO.